### PR TITLE
fix: :art: Refactored some code and fixed testing bugs

### PR DESCRIPTION
**Pull request by rebelchris**
Fixed testing issues and cleanup code by doing small refactors.

- Text error on blocked page
- Styling issue for search rows
- Added new hook for tagContext state

### CODE REVIEW

File: packages/shared/src/contexts/BootProvider.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const BootDataProvider = ({
 
   useRefreshToken(bootData?.accessToken, refetch);
   const { user, updateUser, loadedFromCache } = useCacheUser(bootData?.user);
-  const alertContext = useAlertContext(null);
+  const alertContext = useAlertContext();
```
Comment (idoshamun) [inline packages/shared/src/contexts/BootProvider.tsx:94]: Shouldn't we provide here `bootData?.alerts`?

**Review by idoshamun — CHANGES_REQUESTED** [review]

File: packages/shared/src/contexts/BootProvider.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/contexts/BootProvider.tsx]: Yep 100% 
Didn't realise we had this available already, modified now

**Comment by rebelchris** [comment]
@idoshamun Nice spot, added the actual boot data now

**Review by idoshamun — APPROVED** [review]
🚨 🚨 🚨 🚨 🚨

**Pull request merged**